### PR TITLE
OSDOCS-6698: Updated the TP tables in the OCP 4.14 RN doc

### PR DESCRIPTION
**Pull request by dfitzmau**
[OSDOCS-6729](https://issues.redhat.com/browse/OSDOCS-6729)

Version(s):
4.14

Link to docs preview:
* [Technology Preview features](https://66382--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes#ocp-4-14-technology-preview)
* [Deprecated and Removed features](https://66382--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes#ocp-4-14-deprecated-removed-features)

QE review:
- [ ] QE not required

Additional information:
Key update items sent by JU Lim (PM):

- Table 11 - CSI Google Filestore Driver Operator goes GA in OCP 4.14.  See [OCPSTRAT-498](https://issues.redhat.com//browse/OCPSTRAT-498) (CSI operator for Google Cloud File (GA)) for details.
- Table 12 - GCP Tagging TP in OCP 4.14.
- Table 12 - Static IP assignment with vSphere IPI is TP in OCP 4.14
- Table 12 - add OpenShift on Oracle Cloud Infrastructure Dev Preview in OCP 4.14
- Table 21 - HCP on BM is GA in OCP 4.14
- Table 21 - HCP on OpenShift Virtualization is GA in OCP 4.14

### CODE REVIEW

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -1449,41 +1447,55 @@ In the following tables, features are marked with the following statuses:
...
+|GCP Confidential VMs
+|Not Available
 |Technology Preview
 |General Availability
-|General Availability
 
-|Mount shared entitlements in BuildConfigs in RHEL
-|Technology Preview
-|Technology Preview
+|GCP Tagging
```
Comment (jianli-wei) [inline release_notes/ocp-4-14-release-notes.adoc:1465]: Suggest to replace "GCP Tagging" with "User-defined labels and tags for Google Cloud Platform", thanks!

**Review by jianli-wei — CHANGES_REQUESTED** [review]

File: release_notes/ocp-4-14-release-notes.adoc (hunk 1 above)
Comment (dfitzmau) [reply release_notes/ocp-4-14-release-notes.adoc]: Thanks! Updated.

**Comment by JoeAldinger** [comment]
networking tables are good to go https://issues.redhat.com/browse/OSDOCS-6733

File: release_notes/ocp-4-14-release-notes.adoc [hunk 2]
```diff
@@ -1777,21 +1783,20 @@ In the following tables, features are marked with the following statuses:
...
 
-|Hosted control planes for {product-title} on bare metal
+|Hosted control planes for {product-title} on Amazon Web Services (AWS)
 |Technology Preview
 |Technology Preview
 |Technology Preview
 
-|Hosted control planes for {product-title} on Amazon Web Services (AWS)
-|Technology Preview
+|Hosted control planes for {product-title} on bare metal
```
[the remark this answers is not in the document]
Comment (lahinson) [reply release_notes/ocp-4-14-release-notes.adoc:1791]: Related Jira: https://issues.redhat.com/browse/OSDOCS-8215

File: release_notes/ocp-4-14-release-notes.adoc
Comment (lahinson) [inline release_notes/ocp-4-14-release-notes.adoc:1]: @dfitzmau On line 703, please comment out the following section, which will be updated in mid-November after hosted control planes is GA:
```
[id="ocp-4-14-hcp"]
=== Hosted control planes (Technology Preview)
```

Comment (dfitzmau) [reply release_notes/ocp-4-14-release-notes.adoc:1]: Hi @lahinson . I commented out this empty section.

Comment (lahinson) [reply release_notes/ocp-4-14-release-notes.adoc:1]: @dfitzmau Thank you!

File: release_notes/ocp-4-14-release-notes.adoc [hunk 3]
```diff
@@ -1172,6 +1174,23 @@ In the following tables, features are marked with the following statuses:
...
+|`--include-local-oci-catalogs` argument
+|Not Available
+|General Availability
+|Removed
+
+|`--use-oci-feature` argument
+|General Availability
+|Deprecated
+|Removed
+|====
```
Comment (snarayan-redhat) [inline release_notes/ocp-4-14-release-notes.adoc:1192]: To be consistent with the usage across document, you could reword this as
`--include-local-oci-catalogs` parameter for `oc mirror`
`--use-oci-feature` parameter for `oc mirror`

Comment (dfitzmau) [reply release_notes/ocp-4-14-release-notes.adoc:1192]: Thanks. Updated!

**Comment by dfitzmau** [comment]
> @dfitzmau In table 21, hosted control planes needs to be listed as TP until mid-November. I have a separate Jira and PR to update the table in mid-November. In the attached screen shot highlighted the two rows that need to be changed to "Technology Preview". Screenshot 2023-10-23 at 9 12 22 AM

Thanks, @lahinson . Updated!

**Review by lahinson — APPROVED** [review]
Thanks @dfitzmau!

**Review by lpettyjo — COMMENTED** [review]
Otherwise, LGTM!

File: release_notes/ocp-4-14-release-notes.adoc [hunk 4]
```diff
@@ -1601,14 +1624,13 @@ In the following tables, features are marked with the following statuses:
 |====
```
[the remark this answers is not in the document]
Comment (dfitzmau) [reply release_notes/ocp-4-14-release-notes.adoc:1684]: Hi @lpettyjo . Thanks for reviewing the table. I can see the following entries in the raw file. Am I understanding this correctly that you cannot see these entries?

Screenshot from 2023-10-24 08-57-18

**Review by ctomasko — COMMENTED** [review]
Please edit the RHV deprecation notice as follows:
Red Hat Virtualization (RHV) as a host platform for OpenShift Container Platform was deprecated and is no longer supported. RHV will be removed in a future OpenShift Container Platform release.

**Review by sheriff-rh — APPROVED** [review]
LGTM! Merging.
